### ISSUE TITLE
Some info about running on Manjaro

### ISSUE THREAD

**Issue by TheK0tYaRa** [EDCD/EDMarketConnector#486]
On manjaro it's required to use full path instead of 
`~/.steam/steam/steamapps/compatdata/359320/pfx/drive_c/users/steamuser/Saved Games/Frontier Developments/Elite Dangerous`
for it to work. Also the field "Ship" is empty.

**Comment by TheK0tYaRa** [comment]
Currently it's not empty so i suppose it didn't find it right away.

**Comment by D33pfield** [comment]
Yes that will work. I recommend making a symbolic link to /Frontier Developments/Elite Dangerous/ in a more convenient place though. You'll save yourself a ton of headache and typing in the future.

**Comment by Athanasius** [comment]
Re: original comment.  Basically you're saying Manjaro doesn't support `~` to mean "The user's home directory" ?

I assume you're referring to the instructions for "Linux with Steam Play:" at https://github.com/EDCD/EDMarketConnector/wiki/Installation-&-Setup , yes ?

Can one of you give the *precise* alternate instructions, then I'll update the docs.  Thanks.

**Comment by A-UNDERSCORE-D** [comment]
@Athanasius IIRC I had this as well, appeared that ~ is not being expanded.

**Comment by Athanasius** [comment]
Is `$HOME` expanded ?

Mostly I just want a 100% clear set of instructions to put in the Troubleshooting wiki page then this ticket is done with.

**Comment by A-UNDERSCORE-D** [comment]
I would assume not as that would imply that its expanding environment variables. I would state that a fully resolved path is required. Which can be gotten easily with `realpath ~/path/to/elite` or `pwd` in the directory wanted

**Comment by Athanasius** [comment]
What actually happens in terms of errors experienced if you get this wrong?  i.e. what would be a section title in Wiki->Troubleshooting ?

I'd appreciate it if you could edit Troubleshooting, and perhaps Installation & Setup too to add the relevant information.

**Comment by A-UNDERSCORE-D** [comment]
it would be a "typical" directory not found:
```
>>> open("~/.config/EDMarketConnector/EDMarketConnector.ini")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
FileNotFoundError: [Errno 2] No such file or directory: '~/.config/EDMarketConnector/EDMarketConnector.ini'
>>> open("/home/ad/.config/EDMarketConnector/EDMarketConnector.ini")
<_io.TextIOWrapper name='/home/ad/.config/EDMarketConnector/EDMarketConnector.ini' mode='r' encoding='UTF-8'>
```
and similar. But I think the fix here is to actually expand ~ when looking it up is a better idea. Is an oneline change that I can open a PR for

**Comment by Athanasius** [comment]
Please try out [4.0.3-rc3](https://github.com/EDCD/EDMarketConnector/releases/tag/Release%2F4.0.3-rc3) .

NB: If you happened to try any of the earlier 4.0.3-rcX series please **uninstall** EDMC first, then install -rc3.  All your configuration and third-party plugins will be unaffected (assuming you put third-party plugins in the correct place, not the install folder!).

**Comment by Athanasius** [comment]
The fix is in 4.0.3.
